### PR TITLE
feat: #737 fix the scientific domain view

### PR DESCRIPTION
**Pull request by Anok666**

### CODE REVIEW

**Comment by aleksandrahojszyk** [comment]
Double space between scientific domains -> single space

**Comment by Marcelinna** [comment]
Providers
image
Services
image

**Review by agpul — APPROVED** [review]

File: ui/apps/ui/src/app/dictionary/translateDictionaryValue.ts [hunk 1]
```diff
@@ -11,6 +11,18 @@ import { interoperabilityGuidelinesAuthorTypeDictionary } from './interoperabili
...
 import { trainingDomainDictionary } from './trainingDomainDictionary';
 
+function cleanGuidelineProvider(str: string): string {
+  const index = str.indexOf('.');
+  const newValue = str.substring(index + 1).replace(/_+|-+/g, ' ');
+  return newValue;
+}
+
+function cleanValueTypeForProvider(valueType: string): string {
+  const index = valueType.indexOf('>');
```
Comment (Michal-Kolomanski) [inline ui/apps/ui/src/app/dictionary/translateDictionaryValue.ts:21]: the assumption that data is not nested more than 2 levels in depth (just one ">" - so: category > subcategory only)
@agpul scientific domains cannot be more nested?

File: ui/apps/ui/src/app/layouts/tags/tags.component.ts [hunk 2]
```diff
@@ -74,4 +87,32 @@ export class TagsComponent implements OnChanges {
     this.activeFilter.emit({ filter, value });
     this.viewPortScroller.scrollToPosition([0, 0]);
   }
+
+  cleanValueTypeForProvider(valueType: string): string {
```
Comment (Michal-Kolomanski) [inline ui/apps/ui/src/app/layouts/tags/tags.component.ts:91]: isn't taht a duplicated code?

File: ui/apps/ui/src/app/dictionary/translateDictionaryValue.ts (hunk 1 above)
Comment (Michal-Kolomanski) [reply ui/apps/ui/src/app/dictionary/translateDictionaryValue.ts:21]: it was confirmed with @agpul that is a safe assumption. I give my approval

**Review by Michal-Kolomanski — APPROVED** [review]

File: ui/apps/ui/src/app/layouts/tags/tags.component.ts (hunk 2 above)
Comment (Anok666) [reply ui/apps/ui/src/app/layouts/tags/tags.component.ts]: done

**Review by Michal-Kolomanski — APPROVED** [review]

**Pull request merged**